### ISSUE TITLE
public master instance: wrong url in email

### ISSUE THREAD

**Issue by fkleedorfer** [researchstudio-sat/webofneeds#1767]
The links in an email sent by ownerblue point to satvm02

**Comment by quasarchimaere** [comment]
everything running on this instance is currently configured for internal useonly as the instance itself does not contain any changes yet, we merely have the nginx configuration running on live (for now) that redirects said url to the master server, once the configuration is done and merged into master we need to remove all the old content too since the existing uris would not be accessible from the "outside"
